crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -267,3 +376,38 @@ impl Consensus {
         false
     }
 }
+#[cfg(test)]
+mod tests {
+    #[warn(unused_imports)]
+    use super::*;
+
+    #[allow(dead_code)]
+    fn get_invalid_tx() {
+        unimplemented!(
+            "
+        Implement function that returns only the invalid transactions from testdata/test_txs.json
+        "
+        )
+    }
+    #[allow(dead_code)]
+    fn get_valid_tx() {
+        unimplemented!(
+            "
+        Implement function that returns only the valid transactions from testdata/test_txs.json
+        "
+        )
+    }
+
+    #[test]
+    fn test_validate_transactions() {
+        unimplemented!("
+        Implement test that uses the verify_block_transactions function to accept valid transactions.
+        ")
+    }
```

Comment:
I did some templates, but seems we have to mock it with arbitrary data.

Reply:
Those inner functions don't require a working chainstate, just parse the data and call it directly.